source/components/text-input/text-input.less

```diff
@@ -0,0 +1,64 @@
+html input,
+textarea {
+  background-color: @gray-0;
+  border: 1px solid @gray-500;
+  border-radius: 2px;
+  color: @gray-800;
+  display: block;
+  font-size: 1rem; // 16px
+  padding: 8px;
+  width: 100%;
+
+  /*
+  https://css-tricks.com/snippets/css/style-placeholder-text/
+  */
+  &::-webkit-input-placeholder { // Chrome
+    color: @gray-600;
+  }
+
+  &::-moz-placeholder { // Firefox 19+
+    color: @gray-600;
+    opacity: 1;
+  }
+
+  &:-moz-placeholder { // Firefox 18-
```

Comment:
We can probably remove this block given that we don't support versions of Firefox that old.